packages/realm/react-native/android/src/main/java/io/realm/react/RealmReactPackage.java

```diff
@@ -40,19 +42,30 @@ public NativeModule getModule(String name, final ReactApplicationContext reactCo
 
     @Override
     public ReactModuleInfoProvider getReactModuleInfoProvider() {
+        Class<? extends NativeModule>[] moduleList =
+                new Class[] {
+                        RealmReactModule.class, RealmReactModule.class,
+                };
```

Comment:
Seems a bit complicated to use an array and loop, since we only have one module class? Is this code copied from somewhere?

Reply:
Not copied but i was trying to reproduce `react-native-reanimated` implementation because their implementation works pretty well

Reply:
Actually yes I think I can simplify it